esx_service/vmdk_ops.py

```diff
@@ -806,21 +806,17 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
     error_info, tenant_uuid, tenant_name = auth.get_tenant(vm_uuid)
     if error_info:
         return err(error_info)
-    if tenant_name == auth_data_const.DEFAULT_TENANT:
-        # for DEFAULT tenant, set default_datastore to vm_datastore
+
+    # if default_datastore is not set for tenant,
+    # default_datastore_url will be set to None
+    error_info, default_datastore_url = auth_api.get_default_datastore_url(tenant_name)
```

Comment:
please update the admin command line message on group creation to something like "vm-group xxx is created. Do not forget to `vm-group vm add` and `vm-group privilege add` to enable better access control" 
(adjust to reflect the correct commands please)